Sources/Logging/Logging.swift

```diff
@@ -653,7 +660,21 @@ extension Logger {
 /// configured. `LoggingSystem` is set up just once in a given program to set up the desired logging backend
 /// implementation.
 public enum LoggingSystem {
-    private static let _factory = FactoryBox(StreamLogHandler.standardOutput)
+    private static let _factory = FactoryBox { label, _ in StreamLogHandler.standardOutput(label: label) }
+    private static let _metadataProviderFactory = MetadataProviderBox(.noop)
```

Comment:
I am a bit confused why we need these two boxes? seems like we are storing the metadata factory twice but maybe that can be simplified?

Reply:
can we get rid of '.noop' since it is optional?

Reply:
Ah yes, we can remove .noop -- thank you